atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableDefinition.java

```diff
@@ -115,6 +121,10 @@ public void rowComponent(String componentName, ValueType valueType) {
 
     public void rowComponent(String componentName, ValueType valueType, ValueByteOrder valueByteOrder) {
         Preconditions.checkState(state == State.DEFINING_ROW_NAME);
+        if (CRITICAL_ROW_TYPES.contains(valueType)) {
```

Comment:
use more efficient log style? log.warn("Row component {} of type {}.", componentName, valueType)